package-lock.json

```diff
@@ -1997,12 +2006,6 @@
       "integrity": "sha1-peZUwuWi3rXyAdls77yoDA7y9RM=",
       "optional": true
     },
-    "jschardet": {
```

Comment:
is this no longer needed?

Reply:
I guess not.  All I did was update `package.json`.  `npm` updated `package-lock.json`.